### DISCUSSION TITLE
Is the number of items in the log configurable?

### DISCUSSION THREAD

**Discussion by johanneskastl** [jj-vcs/jj#5210] [Q&A] [answered]
Hi all,

I found that there are a lot of options for configuring the log behaviour.

In my case, I set the `revsets.logs` to `::` to not stop at the latest immutable. I would like to have `jj log` only show the 10 latest changes/commits, as this is enough most of the time. But I found not way to set `--limit 10` in the config file.

For now I will create an alias

```
[aliases]
l = ["log", "-r", "::", "--limit", "10"]
```

Kind Regards,
Johannes

**Comment by martinvonz** [comment, marked as answer]
Since you don't actually want to see all commit, maybe a better solution is to find a revset that better fits your needs instead of using `::`. You can find the default revset [here](https://jj-vcs.github.io/jj/latest/config/#log). You can replace the 2 there by 10 if you want some more immutable history. Then wrap the whole thing in `latest(..., 10)` to find the latest 10 commits in the set if that's what you want.

**Reply by johanneskastl** [reply]
Oh, I was not aware that you can influence the number of entries with a revset. I'll have a look!

Thanks Martin and happy new year!

**Reply by johanneskastl** [reply]
Thanks a lot, this seems to do the trick. Currently setting the revisions to 8 seems to be my sweet spot!
